advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx

```diff
@@ -0,0 +1,321 @@
+---
+title: 'Using'
+description: 'Walkthrough of example usage scenarios'
+---
+
+The following steps demonstrate Precisely Connect CDC’s Replication functionality with EDB Postgres Advanced Server.
+
+## Configure EDB Postgres Advanced Server as Source Instance in Connect CDC Director
+
+The following steps demonstrate the setup and configuration of EDB Postgres Advanced Server as source instance for Connect CDC Director. 
+
+1. Open Connect CDC Director.
+
+2. Click New option.
+
+3. Right click Hosts, Servers, Tables and select New Host.  
+
+4. Enter information for the Host, Host is the machine where Connect CDC Director is running.
+
+![Create New Host](Images/newhost.png)
```

Comment:
![ConnectCDCHostInfo](https://github.com/EnterpriseDB/docs/assets/106532490/a558b73d-64a4-49bd-b434-483dbe81d8a3)
 Created a new image to show what we are thinking with images

Reply:
done